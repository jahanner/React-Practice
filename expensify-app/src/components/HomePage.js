import React from 'react';


const HomePage = () => (
    <div>
    Expensify Dashboard
    </div>
);

export default HomePage;
