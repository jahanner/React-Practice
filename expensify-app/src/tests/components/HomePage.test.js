import React from "react";
import { shallow } from "enzyme";
import HomePage from "../../components/HomePage.js";

test("Should render HomePage", () => {
  const wrapper = shallow(<HomePage />);
  expect(wrapper).toMatchSnapshot();
});
