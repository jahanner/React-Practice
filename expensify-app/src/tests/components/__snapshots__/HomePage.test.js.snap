// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render HomePage 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
